### ISSUE TITLE
Cannot use globstar (**) in --test-match. 

### ISSUE THREAD

**Issue by nrhoffmann** [microsoft/folio#159]
Given the following directory structure.
```
\---even_more
    \---a
        \---really
            \---nested
                    thing.test.ts
                    thing2.test.ts
```
I cannot use a globstar (`**`) to run all the tests in a subfolder.

----

Doesn't work, but should.
```
npx folio . --test-match even_more/a/**/*.ts
```
----
Works as per https://github.com/microsoft/folio/issues/158.
```
npx folio . --test-match even_more/a/really/nested/*.ts
```

**Comment by nrhoffmann** [comment]
I see that you guys are using [minimatch](https://github.com/isaacs/minimatch). Is this an issue upstream?

**Comment by mxschmitt** [comment]
@nrhoffmann could you try to use it in quotes?

**Comment by nrhoffmann** [comment]
@mxschmitt, double quotes (`"`) work, but single quotes (`'`) don't work.

I closed the issue because for some reason it started to work.

The tl;dr is that I am making a test adapter for folio using [vscode-example-test-adpter](https://github.com/hbenl/vscode-example-test-adapter) and [Test Explorer UI](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer). I am generating these commands programmatically and running them in a child process. While making some updates to that code, it started working. Looking back at the git history, I think the issue was that I was putting the path in single quotes (`'`), which, for some reason, doesn't work.

I would like to be able to enclose the path with single quotes because it is an easy way to [escape specials characters ](https://stackoverflow.com/a/20053121/7711481). This is why I tried that way in the first place.

Oddly enough, I can enclose the regex for the `--grep` command in single quotes.
